rand_jitter/src/lib.rs

```diff
@@ -75,7 +75,10 @@ pub use error::TimerError;
 
 use core::{fmt, mem, ptr};
 #[cfg(feature = "std")]
-use std::sync::atomic::{AtomicUsize, ATOMIC_USIZE_INIT, Ordering};
+use std::sync::atomic::{AtomicUsize, Ordering};
+#[cfg(feature = "std")]
+#[allow(deprecated)]  // Since Rust 1.34, `Atomic*::new` is preferred.
```

Comment:
You've explained the deprecation but not the reason to go against it: better to say "reqiured for compatibility with Rust 1.22" (or < 1.xx)?

Reply:
Apparently this became possible in 1.24: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#stabilized-apis-9

Reply:
I changed it to "Required for compatibility with Rust < 1.24".